SQL/Archive/19.0/2018-02-20_add_unique_into_Config.sql

```diff
@@ -0,0 +1,23 @@
+/*  delete duplicate ConfigID-value pairs */
```

Comment:
this should not be going into 19.0 directory

Reply:
done